dev-requirements.txt

```diff
@@ -53,7 +53,7 @@ sphinxcontrib-jsmath==1.0.1 ; python_version >= "3.7" and python_version < "4.0"
 sphinxcontrib-qthelp==1.0.3 ; python_version >= "3.7" and python_version < "4.0"
 sphinxcontrib-serializinghtml==1.1.5 ; python_version >= "3.7" and python_version < "4.0"
 sqlalchemy-stubs==0.4 ; python_version >= "3.7" and python_version < "4.0"
-tomli==2.0.1 ; python_version >= "3.7" and python_version < "3.11"
+tomli==2.0.1 ; python_version >= "3.7" and python_full_version <= "3.11.0a6"
```

Comment:
Is this intentional change? What's 0a6 ?

Reply:
This is generated automatically. I suspect one of our deps changed their requirements to have that.